.ci/build-platform.yml

```diff
@@ -8,65 +8,15 @@ jobs:
       vmImage: ${{ parameters.vmImage }}
       demands: node.js
     timeoutInMinutes: 120 # This is mostly for Windows
+    variables:
+      ESY__CACHE_INSTALL_PATH: ${{ parameters.ESY__CACHE_INSTALL_PATH }}
+      CACHE_FOLDER: $(Pipeline.Workspace)/cache
     steps: 
-      - powershell: $Env:Path
-        continueOnError: true
-        condition: and(eq(variables['AGENT.OS'], 'Windows_NT'), and(eq(variables['Build.Reason'], 'PullRequest'), and(succeeded(), ne(variables['Build.SourceBranch'], variables['System.PullRequest.TargetBranch']))))
-        displayName: "Print env in powershell"
-      # Needed so that the mingw tar doesn't shadow the system tar. See
```

Comment:
Also note that this here was required when running `export-dependencies` and `import-dependencies` because we can't have mingw tar shadowing the windows tar that we need.